### PR TITLE
FABIO/Change Title on Index.html

### PR DESCRIPTION
**Pull request by FabioDwyer**
Just updating Title on the index.html

### CODE REVIEW

**Comment by cma129** [comment]
👍

**Comment by FabioDwyer** [comment]
I did some extra commits to my branch that are not necessary, but I don't know how to delete commits. The only pull request is the change title one. As soon as this one is given the ok from the group, I'm deleting my branch so I can start a fresh one :-)

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Lit Tracker</title>
```
Comment (jdorocho) [inline public/index.html:27]: Looks good

**Review by jdorocho — APPROVED** [review]

**Comment by cma129** [comment]
great!

**Pull request merged**